src/screen/AddPostScreen.js

```diff
@@ -1,46 +1,41 @@
 import React, { useState } from 'react';
-import {
-  Button, Tile, Icon, Input
-} from 'react-native-elements';
-import { connect } from 'react-redux';
+import { Button, Input } from 'react-native-elements';
+import { useTranslation } from 'react-i18next';
 import Block from '../components/Block';
-import { createPostTitle } from '../actions/PostActions';
+import { sendPost } from '../api/socket';
 
+const AddPostScreen = ({ navigation }) => {
+  const goTo = navigation.navigate;
 
-const AddPostScreen = ({ setTitleAction, navigation }) => {
   const [title, setTitle] = useState();
 
   const [text, setText] = useState();
 
+  const { t } = useTranslation();
+
   return (
     <Block>
       <Input
-        placeholder="Title"
+        placeholder={t('title')}
         value={title}
         onChangeText={value => setTitle(value)}
       />
       <Input
-        placeholder="Text"
+        placeholder={t('text')}
         value={text}
         onChangeText={value => setText(value)}
       />
       <Block ph="5">
         <Button
-          title="Add"
+          title={t('add')}
           onPress={() => {
```

Comment:
Avoiding use arrow functions in onPress property.
` The arrow function returns a new function every time. This causes React to think something has changed in our view, when in fact nothing has.`